tamr_client/project.py

```diff
@@ -55,5 +62,56 @@ def _from_json(url: URL, data: JsonDict) -> Project:
         return mastering_project._from_json(url, data)
     elif proj_type == "CATEGORIZATION":
         return categorization_project._from_json(url, data)
+    elif proj_type == "SCHEMA_MAPPING_RECOMMENDATIONS":
+        return schema_mapping_project._from_json(url, data)
     else:
         raise ValueError(f"Unrecognized project type '{proj_type}' in {repr(data)}")
+
+
+def create(
```

Comment:
This function should be internal-only to encourage users to use specific variants of `create` for their specific project type:

```suggestion
def _create(
```

Reply:
changed